examples/notebooks/EfficientNet_Cifar100_finetuning.ipynb

```diff
@@ -1513,7 +1514,7 @@
     "es_patience = 10\n",
     "es_handler = EarlyStopping(patience=es_patience, score_function=default_score_fn, trainer=trainer)\n",
     "evaluator.add_event_handler(Events.COMPLETED, es_handler)\n",
-    "setup_logger(es_handler._logger)\n",
+    "setup_logger(es_handler.logger)\n",
```

Comment:
@vfdev-5 please, notice, I also fix this line, because with newer api we no longer use `_logger`

Reply:
Thanks @trsvchn ! I think a general clean up of our example would be really nice :)